sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 ## 1.0.0-beta.4 (2021-05-24)
```

Comment:
This should be the date that you plan to release or Unreleased if you don't know yet.

Reply:
I am releasing now.. as soon as I can get a sign off. I already ran the PrepareRelease in earlier PR